presentations/day8.md

```diff
@@ -42,7 +42,7 @@ f.close()
 
 ---
 
-# Use `with open` to automatically close when the scope ends!
+# Use `with open` to automatically close when the scope ends! <!-- Have you talked about scope before this point? -->
```

Comment:
Not in terms of variables existing in and out of scope, but definitely in terms of inside if/else blocks and loop blocks.